book/history/sourdough-history.tex

```diff
@@ -9,11 +9,61 @@ \chapter{The history of sourdough}%
     lessons from the past.
 \end{quoting}
 
-Sourdough has been made since ancient times. The exact origins of fermented
+The story of sourdough bread begins in ancient oceans. These oceans were the
+birthplace of all Earth's life. To understand this timeline, envision Earth's
+history compressed into one year. On this scale, January 1st signifies Earth's
```

Comment:
Not sure exactly how you do that but shall be upper script.

also insécable space between month and day.